rust/main/agents/scraper/migration/src/m20230309_000004_create_table_gas_payment.rs

```diff
@@ -136,6 +155,16 @@ pub enum GasPayment {
     /// Used to disambiguate duplicate payments from multiple payments made in
     /// same transaction.
     LogIndex,
+    /// Domain ID of the chain the payment was made on; technically duplicating
+    /// field Domain, but Domain becomes ambiguous as we add Destination domain as well.
+    Origin,
```

Comment:
is it added to make queries easier to understand, or is there some other benefit as well?

Reply:
Yes, mostly for clarity. Since we are adding `destination` domain field, we migrate field `domain` to be `origin`.